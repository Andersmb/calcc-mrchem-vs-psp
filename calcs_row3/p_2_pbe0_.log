Starting job 5688130 on c1-[21,26],c3-[13,27],c5-22,c10-5 at Fri May 20 04:56:40 CEST 2022

Copying p_2_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying p_2_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688130       p_2_pbe0_        180                                             00:57:30      0:0 
5688130.bat+      batch         24        1   00:00:05          0   00:00:05   00:57:30      0:0 
5688130.ext+     extern        180        6   00:00:00          5   00:00:00   00:57:30      0:0 
5688130.0      mrchem.x        180       30   00:56:53         29   02:53:46   00:57:01      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688130                                                                          
5688130.bat+     19340K          0     19340K        0              0          0 
5688130.ext+          0          5          0        0              5          0 
5688130.0     22742328K         29   8671792K        0             24          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688130                                                                                               
5688130.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688130.ext+        0.00M               5          0.00M            0                5              0 
5688130.0           7.32M              27          6.81M      180.69M                1        116.98M 

Job 5688130 completed at Fri May 20 05:54:11 CEST 2022
