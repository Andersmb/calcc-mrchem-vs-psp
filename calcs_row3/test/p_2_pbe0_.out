                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000047        0.000000        2.557366
    1       P            :        2.411149        0.000000       -0.852455
    2       P            :       -1.205598       -2.088147       -0.852455
    3       P            :       -1.205598        2.088147       -0.852455
---------------------------------------------------------------------------
 Center of mass          :       -0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              30
 Beta electrons          :                                              30
 Total electrons         :                                              60
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000095872e+00
    1    1    a          :                              9.999999917749e-01
    2    1    a          :                              9.999996701378e-01
    3    1    a          :                              1.000000359707e+00
    4    1    a          :                              1.000009902036e+00
    5    1    a          :                              1.000005664959e+00
    6    1    a          :                              9.999993769231e-01
    7    1    a          :                              1.000006797365e+00
    8    1    a          :                              9.999998736149e-01
    9    1    a          :                              9.999995577563e-01
   10    1    a          :                              9.999972205166e-01
   11    1    a          :                              1.000001195514e+00
   12    1    a          :                              9.999938754458e-01
   13    1    a          :                              1.000000892190e+00
   14    1    a          :                              9.999983711765e-01
   15    1    a          :                              1.000001799867e+00
   16    1    a          :                              9.999998794391e-01
   17    1    a          :                              9.999989443255e-01
   18    1    a          :                              1.000000193272e+00
   19    1    a          :                              1.000004225476e+00
   20    1    a          :                              9.999999364400e-01
   21    1    a          :                              9.999999151900e-01
   22    1    a          :                              1.000000052452e+00
   23    1    a          :                              1.000000053642e+00
   24    1    a          :                              9.999999268970e-01
   25    1    a          :                              1.000000002119e+00
   26    1    a          :                              1.000000050076e+00
   27    1    a          :                              1.000000030171e+00
   28    1    a          :                              9.999999725260e-01
   29    1    a          :                              1.000000013529e+00
   30    1    b          :                              1.000000039854e+00
   31    1    b          :                              9.999993892597e-01
   32    1    b          :                              9.999995274621e-01
   33    1    b          :                              1.000000359548e+00
   34    1    b          :                              1.000009301408e+00
   35    1    b          :                              1.000005686746e+00
   36    1    b          :                              9.999996819544e-01
   37    1    b          :                              1.000006767928e+00
   38    1    b          :                              9.999998646705e-01
   39    1    b          :                              9.999995578392e-01
   40    1    b          :                              9.999972164965e-01
   41    1    b          :                              1.000001452289e+00
   42    1    b          :                              9.999939079749e-01
   43    1    b          :                              1.000000880786e+00
   44    1    b          :                              9.999984390122e-01
   45    1    b          :                              1.000001080718e+00
   46    1    b          :                              9.999998794025e-01
   47    1    b          :                              9.999989444135e-01
   48    1    b          :                              1.000000149765e+00
   49    1    b          :                              1.000004358816e+00
   50    1    b          :                              9.999999364400e-01
   51    1    b          :                              9.999999151923e-01
   52    1    b          :                              1.000000052150e+00
   53    1    b          :                              1.000000053642e+00
   54    1    b          :                              9.999999268936e-01
   55    1    b          :                              1.000000002886e+00
   56    1    b          :                              1.000000050076e+00
   57    1    b          :                              1.000000016032e+00
   58    1    b          :                              9.999999743962e-01
   59    1    b          :                              1.000000017409e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)               1352.786887661529
 E-N energy              :            (au)              -3873.042660660672
 Coulomb energy          :            (au)                931.422879997497
 Exchange energy         :            (au)                -22.560648334887
 X-C energy              :            (au)                -69.663324976395
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                323.257409678596
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1681.056866312926
 Nuclear energy          :            (au)                323.257409678596
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.357799456634e+03
                         :      (kcal/mol)             -8.520320160411e+05
                         :        (kJ/mol)             -3.564901957430e+06
                         :            (eV)             -3.694760383516e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.793541077476
    1    1    a          :            (au)                -76.787029468724
    2    1    a          :            (au)                -76.778233938588
    3    1    a          :            (au)                -76.773180937792
    4    1    a          :            (au)                 -6.676736273392
    5    1    a          :            (au)                 -6.670598221918
    6    1    a          :            (au)                 -6.658144243566
    7    1    a          :            (au)                 -6.650806847483
    8    1    a          :            (au)                 -4.812630293209
    9    1    a          :            (au)                 -4.809347756471
   10    1    a          :            (au)                 -4.809313743735
   11    1    a          :            (au)                 -4.807761912234
   12    1    a          :            (au)                 -4.804761119264
   13    1    a          :            (au)                 -4.804094852175
   14    1    a          :            (au)                 -4.796758312495
   15    1    a          :            (au)                 -4.793615538472
   16    1    a          :            (au)                 -4.793326363165
   17    1    a          :            (au)                 -4.792722161976
   18    1    a          :            (au)                 -4.790602689129
   19    1    a          :            (au)                 -4.790011428866
   20    1    a          :            (au)                 -0.966847359479
   21    1    a          :            (au)                 -0.628543177100
   22    1    a          :            (au)                 -0.624342810788
   23    1    a          :            (au)                 -0.622124744725
   24    1    a          :            (au)                 -0.374061191083
   25    1    a          :            (au)                 -0.342678167424
   26    1    a          :            (au)                 -0.339455466952
   27    1    a          :            (au)                 -0.333915480383
   28    1    a          :            (au)                 -0.327889049739
   29    1    a          :            (au)                 -0.326211530307
   30    1    b          :            (au)                -76.793540321732
   31    1    b          :            (au)                -76.787032296572
   32    1    b          :            (au)                -76.778220120862
   33    1    b          :            (au)                -76.773171361708
   34    1    b          :            (au)                 -6.676736068385
   35    1    b          :            (au)                 -6.670600477628
   36    1    b          :            (au)                 -6.658144323193
   37    1    b          :            (au)                 -6.650805032138
   38    1    b          :            (au)                 -4.812628734514
   39    1    b          :            (au)                 -4.809347432282
   40    1    b          :            (au)                 -4.809313538521
   41    1    b          :            (au)                 -4.807766457979
   42    1    b          :            (au)                 -4.804767129737
   43    1    b          :            (au)                 -4.804092534539
   44    1    b          :            (au)                 -4.796758386861
   45    1    b          :            (au)                 -4.793614353545
   46    1    b          :            (au)                 -4.793324711377
   47    1    b          :            (au)                 -4.792722254808
   48    1    b          :            (au)                 -4.790601098691
   49    1    b          :            (au)                 -4.790011229756
   50    1    b          :            (au)                 -0.966846480495
   51    1    b          :            (au)                 -0.628543127471
   52    1    b          :            (au)                 -0.624341881829
   53    1    b          :            (au)                 -0.622122615845
   54    1    b          :            (au)                 -0.374060713878
   55    1    b          :            (au)                 -0.342677321767
   56    1    b          :            (au)                 -0.339455763464
   57    1    b          :            (au)                 -0.333914848924
   58    1    b          :            (au)                 -0.327889162417
   59    1    b          :            (au)                 -0.326209759027
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -792.558545698053
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          7.745967e+00       -1357.799456634330         -1.357799e+03
    1          3.913955e-01       -1364.769699719641         -6.970243e+00
    2          1.132083e-01       -1364.994980076620         -2.252804e-01
    3          6.319928e-02       -1365.033056990360         -3.807691e-02
    4          8.952547e-02       -1365.048413048516         -1.535606e-02
    5          3.462753e-02       -1365.051447592644         -3.034544e-03
    6          1.760117e-02       -1365.052600818013         -1.153225e-03
    7          7.375467e-03       -1365.052762201850         -1.613838e-04
    8          4.686051e-03       -1365.052802533094         -4.033124e-05
    9          2.346192e-03       -1365.052813212925         -1.067983e-05
   10          1.836356e-03       -1365.052820252215         -7.039290e-06
   11          1.252207e-03       -1365.052821925578         -1.673363e-06
   12          6.312798e-04       -1365.052822587658         -6.620799e-07
   13          2.606516e-04       -1365.052822840411         -2.527531e-07
   14          1.714615e-04       -1365.052822926720         -8.630877e-08
---------------------------------------------------------------------------
                      SCF converged in 14 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000047        0.000000        2.557366
    1       P            :        2.411149        0.000000       -0.852455
    2       P            :       -1.205598       -2.088147       -0.852455
    3       P            :       -1.205598        2.088147       -0.852455
---------------------------------------------------------------------------
 Center of mass          :       -0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)               1362.920766409967
 E-N energy              :            (au)              -3895.861948797238
 Coulomb energy          :            (au)                937.139130805982
 Exchange energy         :            (au)                -22.622595712479
 X-C energy              :            (au)                -69.885585311547
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                323.257409678596
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)              -1688.310232605315
 Nuclear energy          :            (au)                323.257409678596
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.365052822927e+03
                         :      (kcal/mol)             -8.565835720717e+05
                         :        (kJ/mol)             -3.583945667874e+06
                         :            (eV)             -3.714497797825e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.240581070297
    1    1    a          :            (au)                -77.240577511829
    2    1    a          :            (au)                -77.240572119178
    3    1    a          :            (au)                -77.240569096690
    4    1    a          :            (au)                 -6.637005536249
    5    1    a          :            (au)                 -6.636840715188
    6    1    a          :            (au)                 -6.636836217994
    7    1    a          :            (au)                 -6.636832069878
    8    1    a          :            (au)                 -4.771845055489
    9    1    a          :            (au)                 -4.771377225853
   10    1    a          :            (au)                 -4.771374020022
   11    1    a          :            (au)                 -4.771370880718
   12    1    a          :            (au)                 -4.770978341532
   13    1    a          :            (au)                 -4.770978222085
   14    1    a          :            (au)                 -4.770887730393
   15    1    a          :            (au)                 -4.770885985702
   16    1    a          :            (au)                 -4.770884459683
   17    1    a          :            (au)                 -4.770777683578
   18    1    a          :            (au)                 -4.770775771436
   19    1    a          :            (au)                 -4.770773874572
   20    1    a          :            (au)                 -0.824268177061
   21    1    a          :            (au)                 -0.556878755012
   22    1    a          :            (au)                 -0.556877052166
   23    1    a          :            (au)                 -0.556868684883
   24    1    a          :            (au)                 -0.363308109752
   25    1    a          :            (au)                 -0.310032216034
   26    1    a          :            (au)                 -0.310028658729
   27    1    a          :            (au)                 -0.310023412048
   28    1    a          :            (au)                 -0.279059794781
   29    1    a          :            (au)                 -0.279056839759
   30    1    b          :            (au)                -77.240581075197
   31    1    b          :            (au)                -77.240577511505
   32    1    b          :            (au)                -77.240572120110
   33    1    b          :            (au)                -77.240569094845
   34    1    b          :            (au)                 -6.637005535785
   35    1    b          :            (au)                 -6.636840715570
   36    1    b          :            (au)                 -6.636836217692
   37    1    b          :            (au)                 -6.636832068492
   38    1    b          :            (au)                 -4.771845055754
   39    1    b          :            (au)                 -4.771377225748
   40    1    b          :            (au)                 -4.771374021295
   41    1    b          :            (au)                 -4.771370879878
   42    1    b          :            (au)                 -4.770978341645
   43    1    b          :            (au)                 -4.770978221006
   44    1    b          :            (au)                 -4.770887732174
   45    1    b          :            (au)                 -4.770885984569
   46    1    b          :            (au)                 -4.770884457479
   47    1    b          :            (au)                 -4.770777682909
   48    1    b          :            (au)                 -4.770775771528
   49    1    b          :            (au)                 -4.770773872952
   50    1    b          :            (au)                 -0.824268174687
   51    1    b          :            (au)                 -0.556878758283
   52    1    b          :            (au)                 -0.556877049531
   53    1    b          :            (au)                 -0.556868677126
   54    1    b          :            (au)                 -0.363308111781
   55    1    b          :            (au)                 -0.310032216293
   56    1    b          :            (au)                 -0.310028662764
   57    1    b          :            (au)                 -0.310023407400
   58    1    b          :            (au)                 -0.279059788873
   59    1    b          :            (au)                 -0.279056837213
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -794.218250558677
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000004       -0.000002        0.000014
 Magnitude               :            (au)                        0.000015
                         :         (Debye)                        0.000038
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000004       -0.000002        0.000014
 Magnitude               :            (au)                        0.000015
                         :         (Debye)                        0.000038
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 56m 54s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
